eox_nelp/utils.py

```diff
@@ -78,3 +83,39 @@ def is_valid_national_id(national_id, raise_exception=False):
         )
 
     return check_national_id
+
+
+def extract_course_id_from_string(string):
+    """This return a sub-string that matches the course_ir regex
+
+    Arguments:
+        string: This is a string that could contains a sub-string that matches with the course_id form.
+
+    Returns:
+        course_id <string>: Returns course id or an empty string.
+    """
+    matches = re.search(COURSE_ID_REGEX, string)
+
+    if matches:
+        return matches.group()
+
+    return ""
+
+
+def get_course_from_id(course_id):
+    """
+    Get Course object using the `course_id`.
+
+    Arguments:
+        course_id (str) :   ID of the course
+
+    Returns:
+        Course
+    """
+    course_key = CourseKey.from_string(course_id)
+    course_overviews = CourseOverview.get_from_ids([course_key])
+
+    if course_overviews:
+        return course_overviews[0]
```

Comment:
Here I was testing and found like an error
![Peek 2023-11-17 16-33](https://github.com/eduNEXT/eox-nelp/assets/51926076/22d03b2c-c191-4a10-887b-d7bf011a38ef)

Reply:
Actually, this seem not return a list...
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/b39210cc-ec98-4556-b460-fcb7eec6c5a6)

Reply:
https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content/course_overviews/models.py#L438

Reply:
copy of https://github.com/openedx/event-routing-backends/blob/master/event_routing_backends/helpers.py#L138